fvp.mk

```diff
@@ -12,11 +12,34 @@ OPTEE_OS_PLATFORM = vexpress-fvp
 
 include common.mk
 
+################################################################################
+# Variables used for TPM configuration.
+################################################################################
+BR2_ROOTFS_OVERLAY = $(ROOT)/build/br-ext/board/fvp/overlay
+BR2_PACKAGE_FTPM_OPTEE_EXT_SITE ?= $(CURDIR)/br-ext/package/ftpm_optee_ext
+BR2_PACKAGE_FTPM_OPTEE_PACKAGE_SITE ?= $(ROOT)/ms-tpm-20-ref
+
+# The fTPM implementation is based on ARM32 architecture whereas the rest of the
+# system is built to run on 64-bit mode (COMPILE_S_USER = 64). Therefore set
+# BR2_PACKAGE_FTPM_OPTEE_EXT_SDK manually to the arm32 OPTEE toolkit rather than
+# relying on OPTEE_OS_TA_DEV_KIT_DIR variable.
+BR2_PACKAGE_FTPM_OPTEE_EXT_SDK ?= $(OPTEE_OS_PATH)/out/arm/export-ta_arm32
+
+BR2_PACKAGE_LINUX_FTPM_MOD_EXT_SITE ?= $(CURDIR)/br-ext/package/linux_ftpm_mod_ext
+BR2_PACKAGE_LINUX_FTPM_MOD_EXT_PATH ?= $(LINUX_PATH)
+DEFCONFIG_TPM_MODULE ?= --br-defconfig build/br-ext/configs/linux_ftpm
+
+DEFCONFIG_TSS ?= --br-defconfig build/br-ext/configs/tss
 
 ################################################################################
 # Paths to git projects and various binaries
 ################################################################################
+USE_MEASURED_BOOT	?= 0
```

Comment:
1. I'd rather use y/n for consistency with other makefiles (`DEBUG` is the only exception using 1/0).
2. How about making this shorter: `MEASURED_BOOT`? (it is the name used by TF-A)

Reply:
Done